components/server/src/external/database/reports.py

```diff
@@ -140,3 +140,18 @@ def latest_reports_overview(database: Database, max_iso_timestamp: str = "") ->
 def report_exists(database: Database, report_uuid: ReportId):
     """Return whether a report with the specified report uuid exists."""
     return report_uuid in database.reports.distinct("report_uuid")
+
+
+def latest_report_for_uuids(all_reports: list[Report], *uuids: list) -> list[Report]:
```

Comment:
`*uuids: list` means that every uuid is a list I think? Also, there is a type for the union of ReportId, etc. so you can write:
```suggestion
def latest_report_for_uuids(all_reports: list[Report], *uuids: ItemId) -> list[Report]:
```